tests/cephfs/cephfs_utils.py

```diff
@@ -60,7 +60,7 @@ def _setup_crefi(node):
         # clone crefi repository
         node.exec_command(
             cmd="cd /home/cephuser/; git clone {}".format(
-                "https://github.com/vijaykumar-koppad/Crefi.git"
+                "https://github.com/yogesh-mane/Crefi.git"
```

Comment:
why have you used your fork here. I saw the change in your fork. you have removed `import xattr`. Try giving this change to the upstream repo `(https://github.com/vijaykumar-koppad/Crefi.git)`. Or else yuo will have to keep maintaining your fork always.

Reply:
Hi @yogesh-mane 

When we had last discussion with Hemanth Kumar and @rakeshgm, there was few talks on replacing smallfile over crefi. 
Is there any change of plan here?

Reply:
i guess this is not getting merged any time sooner: https://github.com/vijaykumar-koppad/Crefi/pull/5/files. 
@yogesh-mane fall back to your fork. lets merge it ASAP

Reply:
Hi @rakeshgm , I have tested these changes.
4.2-rhel-7 logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1605962194616
4.2-rhel-8 logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1606131196375